spacy/pipeline/functions.py

```diff
@@ -65,3 +69,88 @@ def merge_subtokens(doc: Doc, label: str = "subtok") -> Doc:
         for span in spans:
             retokenizer.merge(span)
     return doc
+
+
+default_long_token_splitter_config = """
+[config]
+long_token_length = 25
+split_length = 10
+"""
+
+DEFAULT_LONG_TOKEN_SPLITTER_CONFIG = Config().from_str(
+    default_long_token_splitter_config
+)["config"]
+
+
+@Language.factory(
+    "long_token_splitter",
```

Comment:
We could just call it "token splitter"? I mean, if you want to use it to split short tokens, why not 😅

```suggestion
    "token_splitter",
```